src/test/java/org/broadinstitute/consent/http/resources/DraftResourceTest.java

```diff
@@ -272,12 +272,12 @@ void testUploadAttachmentSuccess() throws SQLException {
     when(draftService.getAuthorizedDraft(any(), any())).thenReturn(new DraftStudyDataset("{}", user));
     DraftInterface draftWithAttachment = new DraftStudyDataset("{}", user);
     draftWithAttachment.addStoredFile(mock(FileStorageObject.class));
-    when(draftService.addAttachments(any(), any(), any())).thenReturn(draftWithAttachment);
+    when(draftService.addAttachments(any(), any(), any())).thenReturn(draftWithAttachment.getStoredFiles().stream().toList());
```

Comment:
I would use `List.copyOf()` to convert a `Set` to a `List`, unless you need a mutable copy of the set.
```suggestion
    when(draftService.addAttachments(any(), any(), any())).thenReturn(List.copyOf(draftWithAttachment.getStoredFiles()));
```